### ISSUE TITLE
Remember terminal tabs per-project

### ISSUE THREAD

**Issue by kzhui125** [zed-industries/zed#7145]
### Describe the feature

When you have 4 terminal tabs opening, then open another folder using `zed .` I expect there is only one terminal tab, not 4.

<img width="604" alt="image" src="https://github.com/zed-industries/zed/assets/11925432/2214b522-d868-4832-8846-29cea6b02825">

**Comment by d1y** [comment]
Yes, it's confusing to me, too. I hope it will only have a terminal after opening.

**Comment by JosephTLyons** [comment]
Ooh, I dont know if anyone on the team has seen this before or not - this is new to me. Nice catch @kzhui125.
